google/cloud/storage/tools/make_jwt_assertion_for_test_data.py

```diff
@@ -0,0 +1,83 @@
+"""Create a JWT assertion for test purposes.
```

Comment:
Please add the shebang line and copyright block.

Reply:
Done.

Reply:
s/Google Inc./Google LLC/